supernode/daemon/mgr/task/manager_util.go

```diff
@@ -24,6 +24,14 @@ func (tm *Manager) addOrUpdateTask(ctx context.Context, req *types.TaskCreateReq
 	}
 	taskID := generateTaskID(taskURL, req.Md5, req.Identifier)
 
+	isUnReachable := false
```

Comment:
I suggest we declare a variable which is named by `isReachable` instead of `isUnReachable `. I think that could be more readable.

Reply:
We store the URL into the `taskURLUnReachableStore`. So I think maybe `isUnReachable` is better when we get it. WDYT?